pipelines/deployer/deployer.yaml

```diff
@@ -120,7 +120,8 @@ generate_namespace_values: &generate_namespace_values
       cd config/${PATH_TO_NAMESPACES}
       echo "creating helm compatible values file from namespace data"
       echo 'namespaces:' > $namespace_values_file
-      for ns in *; do
+      for ns in global-* ((cluster-name))-*; do
```

Comment:
Could you use `params` to pass in the `CLUSTER_NAME` so that:

* `--check-creds` can do it's thing (it's not able to check things in code blocks only in specific places)
* inputs are always declared together

Reply:
Can we make `global` `gsp` instead (or as well)?

Reply:
Addressed both.